src/blocks/intermediary/rules.rs

```diff
@@ -1,10 +1,10 @@
 use ahash::RandomState;
 use hashlink::LinkedHashMap;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 
 use crate::blocks::raw::property::{EnumProperty, PropertyKind};
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
Is Serialize necessary here?

Reply:
Nope! That looks like a mistake. Probably leftover from when I was poking and prodding around